deployment/data-streams/utils/txutil/transaction.go

```diff
@@ -0,0 +1,95 @@
+package txutil
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum"
+	gethtypes "github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+)
+
+// PreparedTx represents a transaction that was prepared but not sent to the chain. This is intended to be
+// either signed and executed directly or bundled into an MCMS operation.
```

Comment:
I highly appreciate these comments and the context they provide. What might look obvious while coding, is often confusing when reading. Thanks!

Reply:
Thank you!